style.css

```diff
@@ -0,0 +1,71 @@
+* {
+  font-family: 'Poppins', sans-serif;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  padding: 20px 40px;
+}
+
+.nav-links {
+  display: none;
+}
+
+.main {
+  background: url(images/header-shapes\ mobile@2x.svg);
```

Comment:
- [ ] Kindly ensure you positioned your background right so hence it does not overlap the text and also it does not repeat itself. 
![image](https://user-images.githubusercontent.com/91486791/235145003-9ff5fdaa-1fab-4544-8b7b-e2c7bf8e5cd4.png)